core/domain/stats_domain.py

```diff
@@ -464,6 +514,11 @@ def __init__(self, answer, interaction_id, answer_group_index,
         self.answer_str = answer_str
 
     def to_dict(self):
+        """Returns the dict of submitted answer.
+
+        Returns:
+            dict(str, *). The submitted answer dict.
```

Comment:
"dict. The submitted answer dict." should be enough. Ditto below.

Reply:
Done!